server.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+const cors = require('cors');
+const port = 5555;
+const server = express();
+
+// middleware
```

Comment:
Nice job using comments to organize your code and annotate it for future reference! 🎉